### DISCUSSION TITLE
[Feature Request] Add filter/search for workflows list (add indicator for manual workflows)

### DISCUSSION THREAD

**Discussion by iamazeem** [community/community#53383] [Actions]
### Select Topic Area

Product Feedback

### Body

For multiple workflows, it becomes quite cumbersome to click on the "**Show more workflows...**" button without having a filter/search box.

Here's a sample list with multiple workflows:

![image](https://user-images.githubusercontent.com/7128328/233565189-48456d3c-30a5-48ce-8365-792690ad67f5.png)

Apart from that, it would be good to have an indicator right next of workflow name which have `workflow_dispatch` (manual trigger) in them. The user won't have to look into the workflow file itself to figure that out.

These two would definitely enhance and improve the UI/UX of GHA workflows.

---

Just found out this similar discussion thread (https://github.com/orgs/community/discussions/45221).
Let me know if I need to edit mine and contain it only to the manually triggered workflows. Thanks!

**Comment by mattip** [comment]
There is also https://github.com/orgs/community/discussions/11831
